man/sd_event_new.xml

```diff
@@ -103,6 +113,17 @@
     thread. All threads have exactly either zero or one default event loop
     objects associated, but never more.</para>
 
+    <para>After allocating an event loop object, add event sources to
+    it with
+    <citerefentry><refentrytitle>sd_event_add_io</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    <citerefentry><refentrytitle>sd_event_add_time</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    <citerefentry><refentrytitle>sd_event_add_signal</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    <citerefentry><refentrytitle>sd_event_add_child</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    or
+    <citerefentry><refentrytitle>sd_event_add_defer</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    and then execute the event loop using
```

Comment:
sd_event_add_post() and sd_event_add_exit() are actually on the same man page as sd_event_add_defer(), hence I chose to omit them, given that they are linked anyway...